app.js

```diff
@@ -3,11 +3,21 @@ const {
   getHealthcheck,
   getTopics,
 } = require("./controllers/controllers");
+const { readFile } = require("fs/promises");
 
 const app = express();
 
 app.use(express.json());
 
+app.get("/api", (req, res) => {
+  readFile("./endpoints.json", "utf8").then((file) => {
+    const endpoints = JSON.parse(file);
+    res.setHeader("Content-Type", "endpoints/json");
+    res.write(JSON.stringify({ endpoints: endpoints }));
+    res.statusCode = 200;
+    res.end();
+  });
```

Comment:
Please do this in the controller :) 
You can also simply require the JSON, you don't need an fs.readFile 👍